app/util/participant_edit.py

```
@@ -1,55 +1,78 @@
+            raise ValueError(ISO_CODE_NOT_FOUND)
+
+    # 1国籍のチームの場合、国名を取得して終了
+    if iso_code != MULTI_COUNTRY_TEAM_ISO_CODE:
+        try:
+            if language:
+                beatboxer_data["country"] = beatboxer_data["Country"]["names"][language]
+            beatboxer_data["iso_alpha2"] = [beatboxer_data["Country"]["iso_alpha2"]]
+            beatboxer_data.pop("Country")
+        except KeyError:
+            if language:
+                raise ValueError(ISO_CODE_COUNTRY_NAMES_OR_ALPHA2_NOT_FOUND)
```

Ruff (0.14.1)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)